site/data/docs/custom-wallet-list.mdx

```diff
@@ -378,6 +378,17 @@ zerionWallet(options: {
 });
 ```
 
+#### Coin98 Wallet
```

Comment:
Is the `Coin98 Wallet` name still valid, or should this be changed?

Reply:
Yes, this name is valid. There is no need to change.